apps/expo/src/app/signin.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Image, Text, TextInput, View } from "react-native";
+import type { ImageSourcePropType } from "react-native";
+
+import FilledButton from "~/components/FilledButtons";
+import Logo from "../../assets/logo.png";
```

Comment:
Should we do this with absolute imports? Not sure what the best method is here but `~` is relative to the `src` folder, so we'd have to do `~/../assets` which feels a little weird

Reply:
I think that what you have is fine, as long as it works I don't think it's too important